src/test/TestResult.java

```diff
@@ -1,12 +1,13 @@
-package test;
-
-import domain.Exercise;
-
-public class TestResult {
-
-    public static void main(String[] args) {
-
-        int R=10;
-        System.out.println("Area of a circle for R="+R+"m is "+Exercise.Calculate(R)+"m"+'\u00B2');
-    }
-}
+package test;
+
+import domain.Exercise;
+
+public class TestResult {
+
+    public static void main(String[] args) {
+
+        int m=1;
+        int n=1;
+        System.out.println("Ackerman's function for number"+n+"and"+m+"is"+Exercise.Calculate(n,m));
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.